tests/Kconfig

```diff
@@ -36,9 +36,11 @@ config COVERAGE
 config TEST_USERSPACE
 	bool "Enable userspace if available"
 	depends on ARCH_HAS_USERSPACE
+	depends on TEST
 	select USERSPACE
 	select APPLICATION_MEMORY
-	default n
+	select DYNAMIC_OBJECTS
```

Comment:
These two lines run afoul of me deep and persistent ignorance of the differences between "depends on" and "select" in kconfig.  I'm just going to nod, but secretly wish there was a way to express this without using both of them at once.